src/index.html

```diff
@@ -2,12 +2,15 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>Dosti App</title>
+  <title>Dosti</title>
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
```

Comment:
this I dont mind so much as the other change I commented on in this file, and I know that angular material ui explicitly suggests doing this. I just want to point out that you don't have to do font imports here, these could be done in a style sheet that you import only once.

Reply:
note: I am explicitly calling out the mixed responsibilities here but **not asking for a change**. Let's actually follow the prescribed pattern, just as you do here.